src/App.css

```diff
@@ -1,38 +1,117 @@
-.App {
+body {
+  margin: 0%;
+}
+
+
+.main-div {
+  display: flex;
+  height: 100vh;
+  border: 0;
+  padding: 0;
+}
+
+
+.left-container {
+  background-color: white;
+  width: 100%;
+}
+
+
+.right-container {
+  background-color: #2FBCF7;
```

Comment:
get the colors from a constant file

Reply:
I think you can't do that on pure CSS right?

Reply:
Oh, I didn't know that